### ISSUE TITLE
Add MissingNo.‐like monster

### ISSUE THREAD

**Issue by ghost** [Tuxemon/Tuxemon#404]
Add MissingNo.‐like monster.

**Comment by Qiangong2** [comment]
You're welcome to create the sprites and add them yourself. However, personally I believe that we have enough Tuxemon in the game :)

**Comment by bitcraft** [comment]
@stas730 Please describe in more detail what you mean.

**Comment by Qiangong2** [comment]
@bitcraft https://bulbapedia.bulbagarden.net/wiki/MissingNo.

**Comment by bitcraft** [comment]
hmmm, spooky.  yeah, i think it would be fun to implement something like this.  to eta, of course, but thank you for the suggestion!

**Comment by MirceaKitsune** [comment]
I totally :+1: this and would love to see it happen! Always a fan of good creepypasta Easter eggs.

For those who don't know exactly what this is about: MissingNo was a fallback monster in very old Pokemon console games. The player is not supposed to be able to receive or encounter them, it only popped up if there was a bug or the game was hacked. I believe someone found a way to skip the starter town without getting their starting Pokemon (by clipping through a wall), so when they encountered a battle without having a Pokemon registered, the code sent out a monster called MissingNo which had a corrupt sprite due to an invalid pointer.

In our case it might make sense to use it in a similar fashion: Normally Tuxemon won't start battles if you don't have at least one monster. However there could be an option to use a fallback monster instead, which is given to the player (temporarily or permanently) if a battle is triggered and you have no monsters on you at that time. Making it something glitched and creepy would surely be a great bonus, especially if the game makes sure you can't normally trigger battles before you have a monster so you'd only do that if you found a mapping bug or hacked the game.

If the developers confirm that they might accept this idea, I can try my hand at creating a glitched sprite based on the existing artwork. I seem to be good at making scary edits of things!

**Comment by Qiangong2** [comment]
@MirceaKitsune good idea! However, since you have to walk through grass at the very beginning, it may not be the best choice. Maybe later on in the game your Tuxemon are taken away and you are attacked by Tuxemon belonging to the Omnichannel? Then a glitched Tuxemon (one that was "experimented" on) comes to your rescue and you use it to battle

**Comment by bitcraft** [comment]
the fun about about the glitched pokemon is because you get them when the
game is broken due to bugs.  so, to be accurate, you'd have to glitch the
game by walking through a wall or something to trigger it.

On Tue, May 1, 2018 at 7:50 PM, ZhongQian TiaoGong <notifications@github.com

**Comment by Sanglorian** [comment]
I think a glitch mon is a fun idea, but I don't think it belongs under issues - could be a fun discussion under Discussions though.
